doc/api/process.md

```diff
@@ -2309,6 +2309,227 @@ import { loadEnvFile } from 'node:process';
+assert.strictEqual(process.env.WITHOUT_EQUAL, undefined);
+```
+
+```mjs
+import assert from 'node:assert';
+import process from 'node:process';
+assert.strictEqual(process.env.BASIC, 'basic');
+assert.strictEqual(process.env.WITHOUT_EQUAL, undefined);
+```
+
+#### Whitespace handling
+
+* Leading and trailing whitespaces around keys and values are ignored unless
+  they are enclosed within quotes.
+
+```bash
+      SPACED_KEY = parsed
+```
+
+```cjs
+const assert = require('node:assert');
+const process = require('node:process');
+assert.strictEqual(process.env.SPACED_KEY, 'parsed');
+```
+
+```mjs
+import assert from 'node:assert';
+import process from 'node:process';
+assert.strictEqual(process.env.SPACED_KEY, 'parsed');
+```
+
+#### Comments
+
+* Lines starting with `#` are treated as comments and ignored.
+* Inline comments (starting with `#` after a value) are ignored, and do not
+  affect parsing of the value.
+
+```bash
+    # COMMENTS=work
+    INLINE_COMMENTS=inline comments # work
+```
+
+```cjs
+const assert = require('node:assert');
+const process = require('node:process');
+assert.strictEqual(process.env.COMMENTS, undefined);
+assert.strictEqual(process.env.INLINE_COMMENTS, 'inline comments');
+```
+
+```mjs
+import assert from 'node:assert';
+import process from 'node:process';
+assert.strictEqual(process.env.COMMENTS, undefined);
+assert.strictEqual(process.env.INLINE_COMMENTS, 'inline comments');
+```
+
+#### Empty values
+
+* Variables with no value assigned (just a key followed by `=`) are set to
+  an empty string.
+
+```bash
+  EMPTY=
+```
+
+```cjs
+const assert = require('node:assert');
+const process = require('node:process');
+assert.strictEqual(process.env.EMPTY, '');
+```
+
+```mjs
+import assert from 'node:assert';
+import process from 'node:process';
+assert.strictEqual(process.env.EMPTY, '');
+```
+
+#### Quoted values
+
+* Single (`'`), double (`"`), and backtick (`` ` ``) quotes are recognized.
+  The content within the quotes is taken as is, including leading and
+  trailing spaces.
+* Quotes within a different quote type are preserved.
+
+```bash
+  FIRST_NAME=' John '
+  MIXED_QUOTES="Say 'Hello!'"
+  BACKTICK_IN_QUOTES="Using `backticks` in double quotes"
+```
+
+```cjs
+const assert = require('node:assert');
+const process = require('node:process');
+assert.strictEqual(process.env.FIRST_NAME, ' John ');
+assert.strictEqual(process.env.MIXED_QUOTES, "Say 'Hello!'");
+assert.strictEqual(process.env.BACKTICK_IN_QUOTES,
+                   'Using `backticks` in double quotes');
+```
+
+```mjs
+import assert from 'node:assert';
+import process from 'node:process';
+assert.strictEqual(process.env.FIRST_NAME, ' John ');
+assert.strictEqual(process.env.MIXED_QUOTES, "Say 'Hello!'");
+assert.strictEqual(process.env.BACKTICK_IN_QUOTES,
+                   'Using `backticks` in double quotes');
+```
+
+#### Multiline values
+
+* Values enclosed in quotes described above that span multiple
```

Comment:
Maybe removed, 'described above'?

Reply:
Nop it's under Quotes section

Reply:
I don't think you need the "described above", as it is implied by the section and usage.